src/components/icon/icons/solidArrowDown.jsx

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import SolidArrowUp from './solidArrowUp';
+
+export default function SolidArrowDown({
+  style: styleProp,
+  ...rest // eslint-disable-line comma-dangle
```

Comment:
Is there a reason to disable the comma dangle for the spread operator? I tried removing it and it doesn't seem to cause any issues, though perhaps I've overlooked it.

Reply:
@mlenser sorry what do you mean?

Reply:
I'm asking about `// eslint-disable-line comma-dangle`. I would expect this to not have the eslint-disable like such:

```
export default function SolidArrowDown({
  style: styleProp,
  ...rest,
})
```
It seemed to work when I made that change, but maybe you saw something else.

Reply:
Yeah cool, will update

Reply:
Nice!